config/webhook/kustomization.yaml

```diff
@@ -14,6 +14,9 @@ patchesJson6902:
     name: k8ssandra-operator-validating-webhook-configuration
     kind: ValidatingWebhookConfiguration
   patch: |-
+    - op: replace
```

Comment:
I don't remember if this is something auto-generated, or I did this by accident and should rever this.

Reply:
No, I added this for you.